<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GstObjectManagePanel</class>
 <widget class="QWidget" name="GstObjectManagePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Object Info</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>State</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QRadioButton" name="voidPendingRadioButton">
            <property name="text">
             <string>VOID PENDING</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="nullStateRadioButton">
            <property name="text">
             <string>NULL STATE</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="readyRadioButton">
            <property name="text">
             <string>READY</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="pausedRadioButton">
            <property name="text">
             <string>PAUSED</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="playingRadioButton">
            <property name="text">
             <string>PLAYING</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeObjectButton">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="objectInfoTreeWidget">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="probesGroupBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Probes</string>
         </property>
         <layout class="QGridLayout" name="gridLayout"/>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="requestPadsGroupBox">
         <property name="title">
          <string>Request pads</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6"/>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="sometimesPadsGroupBox">
         <property name="title">
          <string>Sometimes pads</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Object Properties</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
